src/components/Pagination/Pagination.js

```diff
@@ -113,7 +113,7 @@ class Pagination extends Component {
     return (
       <div
         className={cx(this.props.cssClasses.root, {
-          [this.props.cssClasses.noRefinementRoot]: this.props.isFirstPage,
+          [this.props.cssClasses.noRefinementRoot]: this.props.nbPages <= 1,
```

Comment:
can this be returned by the connector?

Reply:
As stated in the description, I didn't change the connector widget render option because we're not consistent yet in this flavor.